docs/developer-docs/web-apps/verifiable-credentials/issuer.mdx

```diff
@@ -0,0 +1,184 @@
+
+For example, when using [canister signatures](https://internetcomputer.org/docs/current/references/ic-interface-spec#canister-signatures)
+`prepared_context` contains a time stamped and unsigned VC, for which the canister signature will be available through the `get_credential` call.
+:::
+
+### Step 4: Issue the credential (`get_credential`)
+
+Lastly, the issuer API endpoint `get_credential` is used to issue the credential requested by the user. The same checks used to prepare the credential are ran again, with an additional verify step to confirm that `prepared_context` is consistent with `prepare_credential`. If successful, the issuer API returns the signed credential in JWS-format.
+
+An example implementation in Rust can be found below:
+
+```rust
+#[query]
+#[candid_method(query)]
+fn get_credential(req: GetCredentialRequest) -> Result<IssuedCredentialData, IssueCredentialError> {
+    if let Err(err) = authorize_vc_request(&req.signed_id_alias, &caller(), time().into()) {
+        return Result::<IssuedCredentialData, IssueCredentialError>::Err(err);
+    };
+    let prepared_context = match req.prepared_context {
+        Some(context) => context,
+        None => {
+            return Result::<IssuedCredentialData, IssueCredentialError>::Err(internal_error(
+                "missing prepared_context",
+            ))
+        }
+    };
+    let credential_jwt = match String::from_utf8(prepared_context.into_vec()) {
+        Ok(s) => s,
+        Err(_) => {
+            return Result::<IssuedCredentialData, IssueCredentialError>::Err(internal_error(
+                "invalid prepared_context",
+            ))
+        }
+    };
+    let signing_input =
+        vc_signing_input(&credential_jwt, &CANISTER_SIG_PK).expect("failed getting signing_input");
+    let message_hash = vc_signing_input_hash(&signing_input);
+    let sig_result = SIGNATURES.with(|sigs| {
+        let sig_map = sigs.borrow();
+        let certified_assets_root_hash = ASSETS.with_borrow(|assets| assets.root_hash());
+        sig_map.get_signature_as_cbor(
+            &CANISTER_SIG_SEED,
+            message_hash,
+            Some(certified_assets_root_hash),
+        )
+    });
+    let sig = match sig_result {
+        Ok(sig) => sig,
+        Err(e) => {
+            return Result::<IssuedCredentialData, IssueCredentialError>::Err(
+                IssueCredentialError::SignatureNotFound(format!(
+                    "signature not prepared or expired: {}",
+                    e
+                )),
+            );
+        }
+    };
+    let vc_jws =
+        vc_jwt_to_jws(&credential_jwt, &CANISTER_SIG_PK, &sig).expect("failed constructing JWS");
+    Result::<IssuedCredentialData, IssueCredentialError>::Ok(IssuedCredentialData { vc_jws })
+}
+```
+
+## Resources
```

Comment:
Can we add a new section: `## Recommended Conventions`

And we can copy [the paragraphs from the spec](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#recommended-convention-connecting-credential-specification-with-the-returned-credentials).

Reply:
Added section